bip-0119.mediawiki

```diff
@@ -170,21 +170,25 @@ specification for the semantics of OP_CHECKTEMPLATEVERIFY.
 Where
 
     bool CheckDefaultCheckTemplateVerifyHash(const std::vector<unsigned char>& hash) {
+        // note: for anti-DoS, a real implementation *must* cache parts of this computation
+        // to avoid quadratic hashing DoS, including 
```

Comment:
That sentence looks incomplete

Reply:
```suggestion
        // to avoid quadratic hashing DoS all variable length computations must be precomputed
        // including hashes of the scriptsigs, sequences, and outputs. See the section
        // "Denial of Service and Validation Costs" below.
```

Reply:
woops :)